apps/molgenis-components/src/components/forms/RowEdit.vue

```diff
@@ -176,38 +177,44 @@ export default {
     },
     isVisible(column: IColumn) {
       try {
+        if (column.visible === "" && this.columnErrors[column.id] !== "") {
```

Comment:
Would it not be better to have !column.visible && !this.columnErrors[column.id] ?
Because I don't think all errors are initialized on ""